src/utils/apolloClient.ts

```diff
@@ -25,6 +30,27 @@ export const getGiraffeEndpoint = (): string => {
   }
 }
 
+export const getGiraffeWsEndpoint = (): string => {
```

Comment:
copy-paste.. we should make a general function for `...WS_ENDPOINT` and `...ENDPOINT